modules/react/select/stories/Select.stories.mdx

```diff
@@ -104,11 +103,9 @@ The `error` prop may be applied directly to the Select with a value of `Select.E
 
 <ExampleCodeBlock code={Error} />
 
-## Props
+## Component API
 
-Undocumented props are spread to the underlying `<select>` element.
-
-<ArgsTable of={Select} />
+<SymbolDoc name="Select" fileName="/react/" hideDescription />
```

Comment:
This is probably fine and maybe intentional, but FYI this component has `ErrorType` as a subcomponent so it has a table that shows up now.
<img width="702" alt="Screenshot 2023-02-24 at 12 23 51 AM" src="https://user-images.githubusercontent.com/1075861/221118939-537d694f-afaa-4718-a1b4-2410043d3012.png">

Reply:
Yes. It is an enum which are considered an `'object'`. Objects render as a table or an inline object depending on the context. To prevent tables in tables, a table switches the render context to inline

Reply:
I thought about adding a special `enum` type, but we are removing enums altogether.